src/utilities/utilities.ts

```diff
@@ -251,6 +251,36 @@ export function swiftDriverSDKFlags(indirect = false): string[] {
     return indirect ? args.flatMap(arg => ["-Xswiftc", arg]) : args;
 }
 
+/**
+ * Get target flags for swiftc
+ *
+ * @param indirect whether to pass the flags by -Xswiftc
+ */
+export function swiftDriverTargetFlags(indirect = false): string[] {
```

Comment:
Not sure about setting the `target` command line parameter based on the sdk parameter. It seems a little fragile although I don't see a quick and easy way to set it otherwise.

Reply:
I agree that this is a bit fragile, but I wasn't sure of another way to do this without exposing SDK versioning information to the user as well, something that wouldn't necessary 95% of the time because XCode's SDK uses symlinks.

Reply:
What do you think about adding/removing the target parameter from the build arguments setting when you choose the platform? It would be clearer to the user what is happening. The current version is adding hidden arguments which might confuse the user. I don't have an issue with exposing the underlying details to the user

Reply:
If we move to using destination files that'll make this a lot simpler